Search_SortedArr_UnknownSize.java

```diff
@@ -0,0 +1,29 @@
+// Time Complexity : O(log (m*n))
```

Comment:
Kindly explain what m and here is

Reply:
m - to find the range (low and high) - first while loop
n - the binary search - second while loop